docs/INSTALL.md

```diff
@@ -19,6 +19,7 @@ Copyright 2019 Intel Corporation
 
 You need the following dependencies.
 
+* By default, in docker_startup.sh it is allocating CPU 12-13, So make sure that you make changes according to the availabe CPUs in your System.
```

Comment:
I think the proper place for this change is in the [installation](https://github.com/omec-project/upf/blob/master/docs/INSTALL.md#installation) section (below) and it should be something like this:

```diff
diff --git a/docs/INSTALL.md b/docs/INSTALL.md
index 39abb84..6da1820 100644
--- a/docs/INSTALL.md
+++ b/docs/INSTALL.md
@@ -142,6 +142,9 @@ To configure/install the UPF in DPDK mode, the following changes are required:
     ./scripts/docker_setup.sh
     ```
 
+    Note: By default, docker_startup.sh uses/sets 2 CPUs for the BESS container (`--cpuset-cpus=12-13`),
+    so, if needed, accordingly adjust `--cpuset-cpus` to the available CPUs in your System.
+
 2. Insert rules into PDR and FAR tables
 
     Use gRPC sim mode to directly install PFCP forwarding rules via gRPC API (works only for BESS)
```

Reply:
Cool

Reply:
Are you going to update the PR with the suggested changes? Or would you like me to take care of that?

Reply:
Oh, Sorry for the inconvenience. I have raised the Pull Request for the same.